content/rc/how-to/creating-aws-user-redis-enterprise-vpc.md

```diff
@@ -9,14 +9,42 @@ aliases: /rv/how-to/creating-aws-user-redis-enterprise-vpc/
 In most Redis Cloud deployments, the infrastructure of your Redis Cloud subscriptions on AWS is created in dedicated AWS accounts that we manage for you.
 In Redis Cloud Ultimate, you can choose to have this infrastructure on your own AWS accounts.
 You'll want these accounts to be separate from any AWS application accounts,
-and you'll need to create some dedicated IAM entities to let us manage the infrastructure for you.
+and you'll need to create some dedicated IAM entities and then configure your Redis Cloud Account with the necessary details.
+
+The configuration can either be done 'by hand', following the instructions in [Create and Edit a Cloud Account for Redis Cloud Ultimate 
+](rc/how-to/view-edit-cloud-account.md), or via the Cloud API, in which case you'll use the [create cloud account using POST](https://api.redislabs.com/v1/swagger-ui.html#/Cloud%20Accounts/createCloudAccountUsingPOST) call.
 
 In the new AWS account, you need to create:
 
 - An **instance role**
-- A user with an **access key**
+- A user with an **access key** and a password
```

Comment:
@TobyHFerguson 
To be precise, API access consists of an Access Key (_AWS Access Key ID_) and a matching Secret Key (_AWS Secret Access Key_). The first one is an identifier and the second one is the actual secret - there isn't really a password involved here.

For console access, there's a username and password.

Reply:
@TobyHFerguson How about "A user with access keys"?

Reply:
The point is that they must have at least one, whereas the implication by your use of the plural is something different. So I'd prefer to stick with my original statement, which was sufficient and necessary.